test/Microsoft.AspNetCore.Identity.EntityFrameworkCore.InMemory.Test/project.json

```diff
@@ -23,24 +24,11 @@
           "version": "1.0.0-*",
           "type": "platform"
         },
-        "moq.netcore": "4.4.0-beta8",
-        "System.Diagnostics.Process": "4.1.0-*",
         "System.Diagnostics.TraceSource": "4.0.0-*"
       },
-      "imports": [
-        "dnxcore50",
-        "portable-net451+win8"
-      ]
+      "imports": "dotnet5.4"
```

Comment:
The Moq package is referencing a bad build of Castle.Core that supports dotnet5.4. We can do a scan of our imports at a later point when they update their dependency.